cypress/e2e/billing.cy.ts

```diff
@@ -24,8 +24,7 @@ describe('Billing', () => {
             cy.get('[data-attr=unsubscribe-reason-survey-textarea]').type('Product analytics')
             cy.contains('.LemonModal .LemonButton', 'Unsubscribe').click()
 
-            cy.window().then((win) => {
-                const events = (win as any)._cypress_posthog_captures
+            cy.pollUntilPresent('survey sent').then((events) => {
```

Comment:
in an e2e test we can't be sure that the events have been emitted and are ready for reading so we need to poll for them